docs/configuration/index.md

```diff
@@ -749,6 +749,30 @@ We provide the following variables:
 (*string*)  Specifies the SIG (special interest group). This option is used
 during cloning of the dist-git repos that are maintained in the SIG namespaces.
 
+#### sync_test_job_statuses_with_builds
```

Comment:
I would write down when an user needs to put this key to `false`. I think it is not straightforward guessing it.